capture-test-execution-system-properties/gradle-test-execution-system-properties.gradle

```diff
@@ -1,10 +1,12 @@
+import java.nio.charset.StandardCharsets
```

Comment:
Is this used anywhere?

Reply:
Yes, it's used line 30 https://github.com/gradle/gradle-enterprise-build-config-samples/pull/138/files#diff-1a883bc084e7cd9bf2bd1f6889e5e0cd8f46f79dc9709bfcd37c10ddfe581f26R30